apis/chm/v1alpha1/types.go

```diff
@@ -0,0 +1,141 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
```

Comment:
I don't think we ever actually use `// +genclient`? Should we add client generation steps to the readme and/or the generated client to the PR?

Reply:
yes, I removed the +genclient